components/Common/Button/index.stories.tsx

```diff
@@ -0,0 +1,15 @@
+import type { Meta as MetaObj, StoryObj } from '@storybook/react';
+import Button from './';
+
+type Story = StoryObj<typeof Button>;
+type Meta = MetaObj<typeof Button>;
+
+export const Default: Story = {
+  args: {
+    variant: 'primary',
+    children: 'Button',
+    disabled: false,
+    special: false,
```

Comment:
Shouldn't this be removed?

Reply:
oh yeah I forgot about that, I was so occupied with the styles